configs/permissions/cost-management.json

```diff
@@ -1,8 +1,10 @@
 {
     "aws.account": ["*"],
```

Comment:
the UI is going to filter out "*" and only show granular resource types and operations, so we'll also need to see those operations defined for each of these resource types. For example, 

` "aws.account": ["*" , "read"],`